lux/extensions/smtp/views.py

```diff
@@ -33,12 +29,16 @@ def post(self, request):
         form = ContactForm(request, data=data)
         if form.is_valid():
             email = request.app.email_backend
-            subject = 'Message from %s' % request.config['APP_NAME']
-            email.send_mail(to=form.cleaned_data['email'],
-                            subject=subject,
-                            message=form.cleaned_data['body'])
-            data = dict(success=True,
-                        message="Message sent")
```

Comment:
Thanks for patch.
I think we want to send an email confirming we have received the message too.
Something from a template.
```
subject: Enquiry received, thank you.
```
and
```
body: We have received your enquiry and we will be incontact with you shortly... and bla bla bla
```